### PR TITLE
Dev/kbroch/reuse v3 compliance

### PR DESCRIPTION
**Pull request by ghost**
related to #243

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: © 2023 Rivos Inc.
```
Comment (glg-rv) [inline .cargo/config.toml:1]: Apparently the Unicode character for Copyright is not needed if SPDX-FileCopyrightText is used. Can we get rid of Copyright symbol? Might cause problem with some editors/terminal combinations.

Comment (kbroch-rivosinc) [reply .cargo/config.toml]: done, replaced with ascii version

File: Cargo.lock [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2023 Rivos Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
```
[the remark this answers is not in the document]
Comment (kbroch-rivosinc) [reply Cargo.lock:4]: I can just dep5 these as well then.

**Comment by JackWolfard** [comment]
Some Copyright holder information is missing from the Apache license. See #248 for details and a fix.

**Comment by abrestic-rivos** [comment]
@kbroch-rivosinc can you pull in @JackWolfard's fix into this PR? LGTM otherwise.

**Review by glg-rv — COMMENTED** [review]
Was confused by adding the '(c)' that was not in the spec, but now, provided other people comments, LGTM.

**Review by abrestic-rivos — APPROVED** [review]

**Pull request merged**